lang/test/org/partiql/lang/eval/io/DelimitedValuesTest.kt

```diff
@@ -14,13 +14,16 @@
 
 package org.partiql.lang.eval.io
 
+import org.partiql.lang.eval.io.DelimitedValues.ConversionMode.NONE
+import org.partiql.lang.eval.io.DelimitedValues.ConversionMode.AUTO
```

Comment:
nit: imports are out of order (i.e. should be with the below `org.partiql` imports)

Reply:
Fixed.